cirq-aqt/cirq_aqt/aqt_sampler.py

```diff
@@ -238,15 +358,15 @@ def _send_json(
         self,
         *,
         json_str: str,
-        id_str: Union[str, uuid.UUID],
+        id_str: str,
```

Comment:
We could use this to distinguish between a noisy and ideal simulator, similarly to the way we do it in Qiskit with the offline simulators. If we do it, we could also add the offline simulators to the `fetch_resources` function.

Reply:
As discussed offline, this was always determined by the simulate_ideal argument and was left like that. The id_str arg is used on the regular sampler to define the label for the circuit, so such a change in function might be counter-intuitive.